src/components/main.tsx

```diff
@@ -782,11 +782,6 @@ class Main extends React.Component<any, MainState> {
       <Footer style={{ textAlign: "center", color: "#969696" }}>
         © {new Date().getFullYear()} GridPlus, Inc. &nbsp;&nbsp; | &nbsp;&nbsp;
         v{APP_VERSION}
-        {constants.ENV === "dev" ? (
```

Comment:
This is the only functional change. It lets the developer know they're in "dev" but there is not difference from "dev" or "prod" so it doesn't seem useful to me.